releases/cockroachcloud-02022020.md

```diff
@@ -0,0 +1,58 @@
+---
+title: What&#39;s New in CockroachCloud
+toc: true
+summary: Additions and changes in CockroachCloud.
+build_for: [cockroachcloud]
+---
+
+## March 02, 2020
+
+In addition to various updates, enhancements, and bug fixes, this beta release includes the following major highlights:
+
+- For all customers who had opted in for CockroachDB upgrades, CockroachCloud clusters running CockroachDB v19.1 have now been upgraded to [v19.1.8](https://www.cockroachlabs.com/docs/releases/v19.1.8.html) and clusters running CockroachDB v19.2 have been upgraded to [v19.2.4](https://www.cockroachlabs.com/docs/releases/v19.2.4.html). To learn how to opt-in for CockroachDB version upgrades for your CockroachCloud clusters, [contact us](https://support.cockroachlabs.com/hc/en-us).
```

Comment:
For customers running 19.1, we haven't kept them up to date. These are all TF clusters.

For customers running 19.2, we have kept them all up to date (except for a few snowflakes).